src/test/scala/mesosphere/marathon/core/task/tracker/impl/InstanceTrackerDelegateTest.scala

```diff
@@ -31,12 +31,12 @@ class InstanceTrackerDelegateTest extends AkkaUnitTest {
   }
 
   "InstanceTrackerDelegate" should {
-    "Provision succeeds" in {
+    "Schedule succeeds" in {
```

Comment:
See below. I don't understand why this test changed.

Reply:
we are just testing that an operation is applied, I believe it does not matter which operation. It does not cover all operations right now anyway. We can definitely do provision but we can also do schedule at this point. I think it does not matter. It's just easier to create scheduled instance so I went that way.

If you see what is it testing, it's just verifying that InstanceTracker received `UpdateContext`. The operation itself is pretty irrelevant.

Reply:
btw. I would like to get rid of `newBuilderWithLaunchedTask` once we merge this one, I think there is no need for that since it's just creating tasks in provisioned state

Reply:
Thanks for explaining. That makes a lot of sense to me now.